Documentation/direct-tools.md

```diff
@@ -68,7 +68,7 @@ After you have created a filesystem as described in the [Shared Filesystem](ceph
 The the other topic you may have mounted the filesystem already in the registry pod. Now we will mount the same filesystem in the toolbox pod.
 This is just a simple way to validate the Ceph filesystem and is not recommended for production Kubernetes pods.
 
-After you have started and connected to the [Rook toolbox](ceph-toolbox.md), proceed with the following commands in the toolbox.
+After you have started and connected to the [Rook toolbox](/cluster/examples/kubernetes/ceph/direct-mount.yaml), proceed with the following commands in the toolbox.
```

Comment:
This section can refer back up to the "Start the direct mount pod" section instead of linking somewhere else

Reply:
Ok, got it.
So I'll write it like

Refer to **Start the Direct Mount pod**  to proceed.
And I'll link the above section on this title?.